src/sagas/projects.js

```diff
@@ -87,6 +90,27 @@ export function* updateProjectSource() {
   yield* saveCurrentProject();
 }
 
+export function* loadAndBeautifyProjectSource() {
+  const state = yield select();
+  const currentProject = getCurrentProject(state);
+  const allFormattedSources = yield all(
+    Reflect.ownKeys(currentProject.sources).map((language) => {
+      const source = currentProject.sources[language];
```

Comment:
Why not use lodash `map(currentProject.sources)` here? That will invoke the iterator with `(source, language)` as arguments.

Reply:
i think i followed suit from errors.js, should probably change that one too